examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -9,13 +9,13 @@
    "cell_type": "markdown",
```

Comment:
<p>maybe this function could go in the libraries, and we can add a small comment here mentioning that we are filtering users and items</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1678/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Moved to datasets/split_utils.py